jobs/ci-test-infra-bazel.sh

```diff
@@ -41,6 +41,6 @@ case "${rc}" in
     *) echo "Unknown exit code: ${rc}" ;;
 esac
 
-./images/pull_kubernetes_bazel/coalesce.py
+/coalesce.py
```

Comment:
fair point, though I think
```bash
export TEST_TMPDIR="/root/.cache/bazel"
```
is already going to fail outside of docker.

long-term I think we want to convert this into a scenario, anyway.